src/main/java/com/raoulvdberge/refinedstorage/integration/forestry/IntegrationForestry.java

```diff
@@ -0,0 +1,64 @@
+package com.raoulvdberge.refinedstorage.integration.forestry;
+
+import com.raoulvdberge.refinedstorage.api.util.IComparer;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.Loader;
+
+public class IntegrationForestry {
+	private static final String ID = "forestry";
+	private static final String QUEEN_BEE = "forestry:bee_queen_ge";
+	private static final String PRINCESS_BEE = "forestry:bee_princess_ge";
+	private static final String GEN_TAG = "GEN";
+	
+    public static boolean isLoaded() {
+        return Loader.isModLoaded(ID);
+    }
+    
+    /**
+     * Determines if any ItemStack in a list contains a Forestry Queen/Princess Bee
+     * and flags it for NBT strip if it contains a "GEN" NBT tag.
+     * 
+     * @param flags	the flags to compare on, see {@link IComparer}
+     * @param items	list of stacks being considered
+     * @return		previous compare flags with COMPARE_STRIP_NBT if any itemstack in list is a 
+     * 				Forestry Queen/Princess Bee with a GEN NBT tag, otherwise only previous compare flags
+     */
+	public static int isItem(int flags, ItemStack... items) {
```

Comment:
This method is kind ambiguous as `isItem` should return a `boolean` indicating whether it is a forestry item or not. Would rather see this method replace with something like a `isBee` method that takes a single `ItemStack`.